### DISCUSSION TITLE
How to upload localized screenshots without using fastlane snapshot

### DISCUSSION THREAD

**Discussion by danilozano** [fastlane/fastlane#17501] [Q&A]
I already have my own localized screenshots for my iOS and Android app, and I was wondering if it was possible to still use fastlane to upload them to the App Store/Play Store if I didn't generate the screenshots using fastlane snapshot. If it is possible, what are the steps to do so? Is it possible to only upload screenshots for an app that is already in the App Store/Play Store? Thanks!

**Comment by ainame** [comment]
>  I was wondering if it was possible to still use fastlane to upload them to the App Store/Play Store if I didn't generate the screenshots using fastlane snapshot

Of course, you can🙂 Have a look at these documents.

https://docs.fastlane.tools/actions/upload_to_app_store/
https://docs.fastlane.tools/actions/upload_to_play_store/

**Comment by rogerluan** [comment]
To complement @ainame 's answer, check out this doc to understand how to structure your files 😃 https://github.com/rogerluan/fastlane/blob/fix-ipad-screenshot-naming/deliver/lib/assets/ScreenshotsHelp
